src/control/server/instance.go

```diff
@@ -43,11 +43,12 @@ import (
 	"github.com/daos-stack/daos/src/control/system"
 )
 
-// IOServerStarter defines an interface for starting the actual
+// IOServerHandler defines an interface for starting and stopping the
 // daos_io_server.
-type IOServerStarter interface {
+type IOServerHandler interface {
```

Comment:
Name suggestion - IOServerRunner
Elsewhere in DAOS we associate "handler" with a service listening and responding to requests, which I don't think this really fits.